addons/account/views/report_invoice.xml

```diff
@@ -245,15 +245,15 @@
                                                 <!-- For desktop -->
                                                 <td name="line_name_td"
                                                     t-att-colspan="(1 if is_product else 6 if display_discount else 5) - line_colspan"
-                                                    t-attf-class="{{'d-none d-md-table-cell' if report_type == 'html' else ''}} {{padding_class}}">
+                                                    t-attf-class="{{'d-none d-md-table-cell' if report_type == 'html' else ''}} border-end-0" t-att-style="padding_style">
                                                     <span t-if="line.name" t-field="line.name" t-options="{'widget': 'text'}">A section title</span>
                                                 </td>
                                                 <!-- For mobile -->
-                                                <td colspan="2" t-attf-class="{{'d-md-none d-table-cell' if report_type == 'html' else 'd-none'}} {{padding_class}}">
+                                                <td colspan="2" t-attf-class="{{'d-md-none d-table-cell' if report_type == 'html' else 'd-none'}} border-end-0" t-att-style="padding_style">
                                                     <span t-if="line.name" t-field="line.name" t-options="{'widget': 'text'}">A section title</span>
                                                 </td>
                                                 <td colspan="1" class="text-end o_price_total">
-                                                    <span t-out="section_subtotal" class="text-nowrap" t-options='{"widget": "monetary", "display_currency": o.currency_id}'/>
+                                                    <span t-out="section_subtotal" class="text-nowrap" t-options='{"widget": "monetary", "display_currency": o.currency_id}' name="price_subtotal_section"/>
```

Comment:
Should have said it earlier, but isn't it kinda fragile to rely on the name for a css target ?  They are mostly meant to ease/provide xpath hooks but not really for style modification (as far as I knew at least).
Why don't you use a dedicated class instead ?